Tests/OTelConventionsTests/OTelAttributesTests.swift

```diff
@@ -0,0 +1,10 @@
+import OTelConventions
+import XCTest
+
+class OTelHttpAttributesTests: XCTestCase {
```

Comment:
Do we want to use Swift Testing instead? *This can also be a follow up task.*

Reply:
Yeah, I've switched the new tests to Swift Testing in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/c1e8493eddac8c210dc89bd4d7d141a9521dc5f1